x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/components/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export { CurationQueries } from './curation_queries';
```

Comment:
👋 Just want to highlight this commit (https://github.com/elastic/kibana/pull/92560/commits/f6dc151fe0a80a97940de137836d28c74ba89414) and note where I've made some significant differences from the Standalone UI.

`<CurationQueries />` and `<CurationQuery />` are almost completely different from ent-search, and require significantly less logic & branching. With @daveyholler's approval and guidance, I've simplified the UX of the curation queries component significantly which results in a massive code cleanup.

The component is used in two places: The Create New view and the Manage Queries modal

## Create curation page

### Before
<img width="710" alt="" src="https://user-images.githubusercontent.com/549407/108944664-0d528c00-7610-11eb-80a7-8df80f5d9686.png">

### After
<img width="808" alt="" src="https://user-images.githubusercontent.com/549407/108944675-10e61300-7610-11eb-8936-0a071064af7b.png">

### Changes

- Removed unnecessary multi-toggle - Davey agreed it does not add any significant confusion to have curation queries default to multiple 
- Allow removing rows - this was functionality present in the modal but hidden for the create view, which feels confusing/inconsistent. I strongly think the ability to control/remove rows should always be present for maximum user agency
- Change `EuiFieldSearch` to `EuiFieldText`: this was done for semantic reasons as an `input type="search"` is simply not the correct field to be using in this scenario - since nothing is actually being searched. This loses us the magnifying glass icon and the ability to clear the input, but IMO the loss there is minimal

## Manage queries modal

### Before

<img width="305" alt="" src="https://user-images.githubusercontent.com/549407/108945294-34f62400-7611-11eb-934d-285717f9ac76.png"> <img width="305" alt="" src="https://user-images.githubusercontent.com/549407/108945253-27409e80-7611-11eb-846a-e5fbbdbb5423.png">

### After
<img width="320" alt="" src="https://user-images.githubusercontent.com/549407/108945337-4ccda800-7611-11eb-91da-96074788acad.png">

### Changes

As you can see, the main UX issue here complicating the 'before' view is the concept of "selecting" the currently active query. It forces the concept of an "editable" vs a "non-editable/selectable" query row.

Davey suggested (and I strongly agree) that it makes **far** more sense to create a select/dropdown on the main curation page itself for switching/selecting the current active query, and have this modal purely in charge of editing/deleting queries. This vastly simplifies & cleans up the component to be able to :

- Always use editable form components, instead of adding any branching clickable interactivity
- Removes the need for a "cancel" button, instead allowing us to simplify to always having the same remove row icon/functionality
- Remove singleton "Save" buttons - instead, we can just save once, at the end of the form 

## Thoughts?

Let me know what y'all think! I think there's potentially some room here as well to use Byron's InlineEditableTable component that he made for Crawler, but I don't want to migrate that over right now, and I think this is a fairly straightforward compromise that is hopefully easy to review (let me know if not) and involves wayyy less code than what's currently in ent-search. I'm definitely open to investigating InlineEditableTable later post-MVP.

Reply:
I think it's a great improvement.